raco/algebra.py

```diff
@@ -1797,6 +1937,28 @@ def scheme(self):
         return None
 
 
+class UntilConvergence(NaryOperator):
```

Comment:
Not really empty since `DoWhile` has a check on having >=2 children (the last one is the stopping condition) while `UntilConvergence` does not. I can make them inherit a common class with that additional check if you prefer.